src/components/navFooter.js

```diff
@@ -0,0 +1,41 @@
+import isamplesLogo from '../images/isampleslogopetal.png';
+
+/**
+ * a function to return navigation bar
+ * @returns jsx component
+ */
+const NavBar = function () {
+  return (
+    <nav className="navbar navbar-expand-lg navbar-light bg-light">
+      <img src={isamplesLogo} alt="iSamples Logo Petal"
+        width="100"></img>
+      <span className="navbar-brand ">Internet of Samples: iSamples</span>
+    </nav>
+  );
+}
+
+/**
+ * a function to return footer bar
+ * @returns jsx component
+ */
+const FooterBar = function () {
+  return (
+    <>
+      <div>&nbsp;</div>
```

Comment:
This is an empty element. If we don't include it, the footer will include the whole content of `solr-faceted-search-react`.